src/runtime/runtime.ts

```diff
@@ -912,9 +902,20 @@ export class ConfiguredRuntime implements Deps, SubscriptionsInterface {
     if (params?.publisherProvidedId) {
       params.publisherProvidedId = this.publisherProvidedId_;
     }
-    const entitlements = await this.entitlementsManager_.getEntitlements(
-      params
-    );
+    const entitlementsResponse =
```

Comment:
I think this would be a little clearer

```suggestion
    const entitlementsPromise =
```

Reply:
Done.